api/controllers/artists.py

```diff
@@ -3,8 +3,15 @@
 from sqlalchemy.orm import Session
 from api.database import get_db
 from api.schemas import ArtistSchema
```

Comment:
(1/4) Could this be moved down to be imported with the other `api.schema` imports?
```suggestion
```